apollo-compiler/src/test/graphql/com/example/arguments_complex/TestQuery.kt

```diff
@@ -42,15 +42,17 @@ data class TestQuery(
   @Transient
   private val variables: Operation.Variables = object : Operation.Variables() {
     override fun valueMap(): Map<String, Any?> = mutableMapOf<String, Any?>().apply {
```

Comment:
Just want to compare this without using `apply` here. I would look like:

```
override fun valueMap(): Map<String, Any?> {
  val map = mutableMapOf<String, Any?>()
  if (episode.defined) map["episode"] = episode.value
  map["stars"] = stars
  return map
}
```

Reply:
well local variable `map` can collide if there is variable with the same name. To be 100% safe we need use `this@`